src/mmw/apps/bigcz/clients/usgswqp/serializers.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from __future__ import (absolute_import,
+                        division,
+                        print_function,
+                        unicode_literals)
+
+from rest_framework.serializers import (CharField,
+                                        DateTimeField,
+                                        ListField,
+                                        Serializer,
+                                        )
+
+from apps.bigcz.serializers import ResourceSerializer
+
+
+class USGSVariableSetSerializer(Serializer):
+    id = CharField()
+    name = CharField()
+    concept_keyword = CharField()
+    site = CharField()
+    wsdl = CharField()
+
+
+class USGSResourceSerializer(ResourceSerializer):
+    details_url = CharField()
+    sample_mediums = ListField(child=CharField())
+    variables = ListField(child=USGSVariableSetSerializer())
+    service_org = CharField()
+    service_orgname = CharField()
+    service_code = CharField()
+    service_url = CharField()
+    monitoring_type = CharField()
+    provider_name = CharField()
+    service_title = CharField()
+    service_citation = CharField()
+    begin_date = DateTimeField()
+    end_date = DateTimeField()
```

Comment:
Other lint issues like this can be discovered by running the `check.sh` script:

```
> ./scripts/check.sh
+ vagrant ssh app -c 'flake8 /opt/app/apps --exclude migrations || echo flake8 check failed'
/opt/app/apps/bigcz/clients/usgswqp/__init__.py:14:36: W292 no newline at end of file
/opt/app/apps/bigcz/clients/usgswqp/models.py:13:80: E501 line too long (84 > 79 characters)
/opt/app/apps/bigcz/clients/usgswqp/models.py:17:46: E127 continuation line over-indented for visual indent
/opt/app/apps/bigcz/clients/usgswqp/models.py:18:46: E127 continuation line over-indented for visual indent
/opt/app/apps/bigcz/clients/usgswqp/models.py:32:43: W292 no newline at end of file
/opt/app/apps/bigcz/clients/usgswqp/search.py:75:25: E251 unexpected spaces around keyword / parameter equals
/opt/app/apps/bigcz/clients/usgswqp/search.py:89:5: F841 local variable 'to_date' is assigned to but never used
/opt/app/apps/bigcz/clients/usgswqp/search.py:90:5: F841 local variable 'from_date' is assigned to but never used
/opt/app/apps/bigcz/clients/usgswqp/search.py:103:42: E128 continuation line under-indented for visual indent
/opt/app/apps/bigcz/clients/usgswqp/search.py:105:80: E501 line too long (103 > 79 characters)
/opt/app/apps/bigcz/clients/usgswqp/serializers.py:37:31: W292 no newline at end of file
flake8 check failed
Connection to 127.0.0.1 closed.
```

Reply:
Thanks! Will fix those.